.aws/src/config/index.ts

```diff
@@ -1,27 +1,23 @@
 const name = 'RecommendationAPI';
-let environment;
-let domain;
-let cacheNodes;
-let cacheSize;
+const domainPrefix = 'recommendation-api';
+const isDev = process.env.NODE_ENV === 'development';
+const environment = isDev ? 'Dev' : 'Prod';
+const isProd : boolean = environment === "Prod";
+const domain = isDev
+  ? `${domainPrefix}.getpocket.dev`
+  : `${domainPrefix}.readitlater.com`;
 
-if (process.env.NODE_ENV === 'development') {
-  environment = 'Dev';
-  domain = 'recommendation-api.getpocket.dev';
-  cacheNodes = 1;
-  cacheSize = 'cache.t3.micro';
-} else {
-  environment = 'Prod';
-  domain = 'recommendation-api.readitlater.com';
-  // aiocache currently does not support data partitioning, so there's little benefit to having more than 1 node.
-  cacheNodes = 1;
-  cacheSize = 'cache.t3.medium';
-}
+// aiocache currently does not support data partitioning, so there's little benefit to having more than 1 node.
+const cacheNodes = isDev ? 1 : 1;
+const cacheSize = isDev ? 'cache.t2.micro' : 'cache.t3.medium';
 
 export const config = {
   name,
   prefix: `${name}-${environment}`,
   circleCIPrefix: `/${name}/CircleCI/${environment}`,
   shortName: 'RECAPI',
+  isProd,
+  isDev,
```

Comment:
This is so minor and I wouldn't block on it, but in general we've been using one or the other of these rather than both. Should we move towards both in other repos?

Reply:
hmm We should have a single one. Let me merge this and then I will followup with a a change in the next one.